glide.yaml

```diff
@@ -0,0 +1,37 @@
+package: github.com/m3db/m3nsch
```

Comment:
Can we add versions to all modules here to pin the modules?  Just so we don't get random upgrades to packages unless we specify so.  Pinning to at least a semver to avoid running glide update and getting a breaking change from packages not meant to be upgraded is important.